olp-cpp-sdk-core/tests/logging/FileAppenderTest.cpp

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <gtest/gtest.h>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <olp/core/logging/Configuration.h>
+#include <olp/core/logging/FileAppender.h>
+#include <olp/core/logging/Log.h>
+#include <olp/core/porting/platform.h>
+
+#ifdef PORTING_PLATFORM_WINDOWS
+#include <stdio.h>
+#define unlink _unlink
+#else
+#include <unistd.h>
+#endif
```

Comment:
Have to use this one for now, as we don't have a simpler way to remove files by relative path.